### PR TITLE
feat(pack): add astro pack

### PR DESCRIPTION
**Pull request by taskylizard**

### CODE REVIEW

File: lua/astrocommunity/pack/astro/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Astro Language Pack
+
+This plugin pack does the following:
+
+- Adds `astro` Treesitter parser
+- Adds `astro` language server
+- Adds `js-debug-adapter` for debugging
+- Adds [nvim-dap-vscode-js](https://github.com/mxsdev/nvim-dap-vscode-js) for debugging
```
Comment (mehalter) [inline lua/astrocommunity/pack/astro/README.md:8]: Should this line be there? It doesn't seem like this pack adds this plugin

**Review by mehalter — CHANGES_REQUESTED** [review]
Does this pack add `nvim-dap-vscode-js`?

File: lua/astrocommunity/pack/astro/README.md (hunk 1 above)
Comment (taskylizard) [reply lua/astrocommunity/pack/astro/README.md]: No it shouldn't be, fixed.

**Review by mehalter — APPROVED** [review]
Thanks! This looks awesome :D

**Comment by taskylizard** [comment]
Cheers! 🥂

**Comment by taskylizard** [comment]
Cheers! 🥂

**Pull request merged**